python/grass/script/core.py

```diff
@@ -1709,44 +1718,38 @@ def create_location(
     desc=None,
     overwrite=False,
 ):
-    """Create new location
+    """Create new project
 
     Raise ScriptError on error.
 
-    :param str dbase: path to GRASS database
-    :param str location: location name to create
-    :param epsg: if given create new location based on EPSG code
-    :param proj4: if given create new location based on Proj4 definition
-    :param str filename: if given create new location based on georeferenced file
-    :param str wkt: if given create new location based on WKT definition
+    :param str path: path to GRASS database
+    :param str name: project name to create
+    :param str mapset: mapset within given project (default: 'PERMANENT')
```

Comment:
Looks like mapset is missing from the arguments, which probably make sense, but it shouldn't be here.